services/web/server/src/simcore_service_webserver/socketio/server.py

```diff
@@ -29,6 +30,7 @@ async def _socketio_server_cleanup_ctx(app: web.Application) -> AsyncIterator[No
         logger=use_logger,
         engineio_logger=False,
         client_manager=server_manager,
+        json=OrJsonAdapter,
```

Comment:
If the serialization is a bottleneck, maybe it would make sense to try to setup a test which checks this 🤔. Just a suggestion

Reply:
@sanderegg and me were discussing about how to do that.